cmd/ocm-backplane/login/login.go

```diff
@@ -204,6 +204,11 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 		"ID":   clusterID,
 		"Name": clusterName}).Infoln("Target cluster")
 
+	//PrintClusterInfo After Login
+	if err := login.PrintClusterInfo(clusterID); err != nil {
```

Comment:
Can we make this function as optional? Either control via a cmd flag or config file

Reply:
I believe it can be pass in as a flag if that's you think is the way to go. Let me see if I can get it done